template-ts/template/src/navigation/MainNavigator.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { NavigationContainer } from '@react-navigation/native'
+import { createStackNavigator } from '@react-navigation/stack'
+
+import HomeScreen from '../screens/HomeScreen'
+
+type MainStackParamList = {
+  HomeScreen: undefined,
```

Comment:
If you want to specify the "empty" state Implicitly, I think it is better to use null here.

Reply:
It's from the lib docs

Reply:
https://reactnavigation.org/docs/typescript#type-checking-the-navigator

Reply:
Also, `null` is kinda for empty value while `undefined` is for the absence of any value.